### PR TITLE
feat(route): add route for easynomad

### PR DESCRIPTION
**Pull request by jiangsong216**
## Involved Issue / 该 PR 相关 Issue

Close #

## Example for the Proposed Route(s) / 路由地址示例

```routes
/easynomad
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [x] New Route / 新的路由
  - [x] Follows [Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [ ] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [ ] Parsed / 可以解析
  - [ ] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/routes/easynomad/joblist.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+import MarkdownIt from 'markdown-it';
+const md = MarkdownIt({
+    html: true,
+    linkify: true,
+});
+import { parseDate } from '@/utils/parse-date';
+
+export const route: Route = {
+    path: '/',
+    categories: ['popular'],
```
Comment (TonyRL) [inline lib/routes/easynomad/joblist.ts:12]: This does not belong to popular

File: lib/routes/easynomad/joblist.ts [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    view: ViewType.Notifications,
+    example: '/easynomad',
+    radar: [
+        {
+            source: ['easynomad.cn'],
+        },
+    ],
+    name: '轻松游牧-远程工作聚集地',
```
Comment (TonyRL) [inline lib/routes/easynomad/joblist.ts:20]: Route's name shouldn't be the same as namespace's name

File: lib/routes/easynomad/joblist.ts [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    maintainers: ['oldj'],
```
Comment (TonyRL) [inline lib/routes/easynomad/joblist.ts:21]:
```suggestion
    maintainers: ['jiangsong216'],
```

File: lib/routes/easynomad/joblist.ts [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+};
+
+async function handler() {
+    const host = `https://easynomad.cn`;
```
Comment (TonyRL) [inline lib/routes/easynomad/joblist.ts:35]:
```suggestion
    const host = 'https://easynomad.cn';
```
Only use template literals when necessary

File: lib/routes/easynomad/joblist.ts [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+    const url = 'https://easynomad.cn/api/posts/list?limit=80&page=1&jobCategory=&contractType=';
```
Comment (TonyRL) [inline lib/routes/easynomad/joblist.ts:36]: Use the default `limit` from the web, which is 15.

File: lib/routes/easynomad/joblist.ts [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+
+    const items = data.map((item) => ({
+        title: item.jobTitle,
+        description: item.descContent ? md.render(item.descContent) : 'No description',
+        pubDate: parseDate(item.jobPublishTime),
+        link: item.url,
+    }));
+
+    return {
+        title: `轻松游牧-远程工作聚合列表`,
```
Comment (TonyRL) [inline lib/routes/easynomad/joblist.ts:51]:
```suggestion
        title: '轻松游牧-远程工作聚合列表',
```

File: lib/routes/easynomad/joblist.ts [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+        description: `支持国内远程的招聘列表，远程全职，远程兼职`,
```
Comment (TonyRL) [inline lib/routes/easynomad/joblist.ts:52]:
```suggestion
        description: '支持国内远程的招聘列表，远程全职，远程兼职',
```

File: lib/routes/easynomad/templates/description.art [hunk 8]
```diff
@@ -0,0 +1,3 @@
+<p>Title: {{title}}</p>
```
Comment (TonyRL) [inline lib/routes/easynomad/templates/description.art:1]: This route didn't use this file at all.

**Comment by jiangsong216** [comment]
@TonyRL Thanks for your review, all issues have been fixed, please check

**Pull request merged**